chia/server/server.py

```diff
@@ -639,6 +639,13 @@ async def wrapped_coroutine() -> Optional[Message]:
                     if response is not None:
                         response_message = Message(response.type, full_message.id, response.data)
                         await connection.send_message(response_message)
+                    elif metadata.reply_types:
```

Comment:
@arvidn this is where we catch the failed request and return a failure message instead of just not responding